src/http.js

```diff
@@ -54,7 +54,7 @@ function shouldDownloadAsText(contentType) {
 }
 
 // Serialize the response, returns a promise with headers and the body part of the hash
-export function serializeRes(oriRes, url, {loadSpec = false} = {}) {
+export function serializeRes(oriRes, url, {loadSpec = false}) {
```

Comment:
We need the default val in case user doesn't pass in an object for the 3rd arg, which would cause an destructuring error.